packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_sidecar/disk.py

```diff
@@ -12,14 +12,14 @@
 
 
 @log_decorator(_logger, level=logging.DEBUG)
-async def free_reserved_disk_space(
+async def delete_reserved_disk(
```

Comment:
did the function really change?
delete_reserved_disk in a file system means that you delete a entire disk.
`free_reserved_disk_space` means that you delete files inside some reserved disk location. Are you sure?